js/tests/unit/scrollspy.js

```diff
@@ -3,6 +3,22 @@ $(function () {
 
   QUnit.module('scrollspy plugin')
 
+  // To keep unit tests consistent with DRY principles
+  function testElementIsActiveAfterScroll (element, target, assert) {
```

Comment:
@cvrebert Shouldn't JSCS have caught this with https://github.com/twbs/bootstrap/blob/master/js/.jscsrc#L11?